RLTest/loader.py

```diff
@@ -70,12 +71,16 @@ def get_functions(self, instance):
 
 
 class TestLoader(object):
-    def __init__(self, filter=None):
+    def __init__(self):
         self.tests = []
-        self.toplevel_filter = filter
-        self.subfilter = None
 
     def load_spec(self, arg):
```

Comment:
If arg can now take multiple types, we should add a type hint here (specifically).

Reply:
arg is never used - this is dead code.